td/tracking/templates/tracking/event_form.html

```diff
@@ -37,6 +37,11 @@ <h2 class="panel-title">New Translation Workshop/Event</h2>
 				<div class="panel-body">
 					<form action="" method="POST" class="form">
 						{% csrf_token %}
+<!-- 						<div class="form-group">
```

Comment:
Commenting out in template is better done with:

```
{% comment %}{% endcomment %}
```

tags.

Unless for some reason you want what you are commenting out to still appear in the rendered HTML source